### PR TITLE
Added accurate choropleth polygon data for prefectures and prefecture boundary highlight on mouseover.

### PR DESCRIPTION
**Pull request by alec-harris**
@reustle 

Hi Shane
I made a small improvement to the map.

Please have a look at my covid tracker (using data from your repo) if you have time.
https://covid-19-18b2d.firebaseapp.com

Screen Shot 2020-07-27 at 10 42 57 am

### CODE REVIEW

File: src/components/OutbreakMap/DrawMapPrefectures.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {
   LEGEND_CLASSES,
 } from "../../data/constants";
 
-const PREFECTURE_JSON_PATH = "static/prefectures.geojson";
+const PREFECTURE_JSON_PATH = "static/prefectures2.geojson";
```
[the remark this answers is not in the document]
Comment (alec-harris) [reply src/components/OutbreakMap/DrawMapPrefectures.js:10]: I think "prefectures-vector.geojson" is probably the best description.

You're welcome, glad you like the changes, if there are any more map related modifications you need, feel free to let me know.

File: src/components/OutbreakMap/DrawMapPrefectures.js [hunk 2]
```diff
@@ -106,20 +107,20 @@ const drawMapPrefectures = (ddb, map, lang) => {
...
-        layout: {},
-        paint: {
-          "line-width": 0.5,
-          "line-color": "#c0c0c0",
-          "line-opacity": 0.5,
-        },
-      },
-      firstSymbolId
-    );
+    // let prefBorderLayer = map.addLayer(
```
[the remark this answers is not in the document]
Comment (alec-harris) [reply src/components/OutbreakMap/DrawMapPrefectures.js:110]: Yes, that block can be deleted.

File: src/components/OutbreakMap/DrawMapPrefectures.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (liquidx) [reply src/components/OutbreakMap/DrawMapPrefectures.js:10]: How about calling it prefectures-smooth.geojson? That seems like what the difference between the original and this one.

File: src/components/OutbreakMap/DrawMapPrefectures.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (liquidx) [reply src/components/OutbreakMap/DrawMapPrefectures.js:110]: Could you remove it and push an update to this PR?

File: src/components/OutbreakMap/DrawMapPrefectures.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (alec-harris) [reply src/components/OutbreakMap/DrawMapPrefectures.js]: Right, ok I've gone ahead and renamed the file to prefectures-smooth.geojson.
I also deleted the original prefectures.geojson file.

File: src/components/OutbreakMap/DrawMapPrefectures.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (alec-harris) [reply src/components/OutbreakMap/DrawMapPrefectures.js]: Ok, I deleted the code.

**Comment by alec-harris** [comment]
Hi Liquidx

Did I send the changes correctly or did I screw something up?
As you've probably noticed it's my first pull request.

Let me know if there is anything else I can do...

**Comment by liquidx** [comment]
Apologies, I had been busy just keeping up with the data changes and I hadn't gotten around to merging this PR. Thanks for the making the changes! 

Merging this in.

**Pull request merged**